packages/plugin-user-agent-enrichment-browser/src/user-agent-enrichment-plugin.ts

```diff
@@ -0,0 +1,43 @@
+import { BrowserConfig, EnrichmentPlugin, Event } from '@amplitude/analytics-types';
+import { CreateUserAgentEnrichmentPlugin, Options } from './typings/user-agent-enrichment-plugin';
+import UAParser from '@amplitude/ua-parser-js';
+
+export const userAgentEnrichmentPlugin: CreateUserAgentEnrichmentPlugin = function (options: Options = {}) {
+  const { osName = true, osVersion = true, deviceManufacturer = true, deviceModel = true } = options;
+
+  let uaResult: UAParser.IResult;
+
+  const plugin: EnrichmentPlugin = {
+    name: '@amplitude/plugin-ua-parser',
```

Comment:
nits: should we keep the name the same as the package name `@amplitude/plugin-user-agent-enrichment-browser` and in logs